gbm-cli/cmd/release/README.md

```diff
@@ -59,5 +60,5 @@ Command used to check the status of any given release:
 **Usage**
 
 ```
-go run main.go release status 1.07.0 
+go run main.go release status 1.07.0
```

Comment:
Took me awhile to review this after failing miserably to succeed performing the testing instructions. Turns out, `1.07.0` is not a valid semver value! 😄 

A preceding zero is invalid. This caused the semver parsing to strip the preceding zero, which resulted in the CLI not locating the PR _containing the preceding zero_.

```suggestion
go run main.go release status 1.7.0
```

Reply:
Oh geez, sorry about that. TIL 1.0X.X is invalid